sources/web/datalab/polymer/modules/GapiManager.ts

```diff
@@ -34,7 +40,8 @@ class GapiManager {
     // Loads the gapi client library and the auth2 library together for efficiency.
     // Loading the auth2 library is optional here since `gapi.client.init` function will load
     // it if not already loaded. Loading it upfront can save one network request.
-    GapiManager._loadClientId()
+    console.log('== loading gapi');
+    return GapiManager._loadClientId()
```

Comment:
I think putting explicit return types on all public async functions is a fine idea. Done (for GapiManager).